kong/plugins/zipkin/schema.lua

```diff
@@ -62,6 +62,8 @@ return {
           { static_tags = { type = "array", elements = static_tag,
                             custom_validator = validate_static_tags } },
           { http_span_name = { type = "string", required = true, default = "method", one_of = { "method", "method_path" } } },
+          { connect_timeout = {type = "number", default = 5000 } },
+          { byte_timeout = {type = "number", default = 10000 } }
```

Comment:
I think it makes more sense to introduce `connect_timeout`, `send_timeout`, `read_timeout`. 
`byte_timeout` may a bit confusing.

ref: https://github.com/openresty/lua-resty-core/blob/master/lib/ngx/balancer.md#set_timeouts

Reply:
Thank you, I have made these amendments.